package/android/.project

```diff
@@ -22,12 +22,12 @@
 	</natures>
 	<filteredResources>
 		<filter>
-			<id>0</id>
+			<id>1673351330942</id>
```

Comment:
is this correct?

Reply:
It seems like the .project file is an Eclipse file that is used by Android Studio when opening the project - nothing to do with the example project. Guess it should be ok to include.